### DISCUSSION TITLE
Trying to Build the source code

### DISCUSSION THREAD

**Discussion by russellmm** [seerge/g-helper#1200] [Q&A] [answered]
I am trying to build the source code with Visual Studio 2022 Pro. I am getting a lot of warnings (see attached log).
It creates a ghelper.dll and a ghelper.exe (but the exe is only 250kb) and does not run.

I have tried building with both the solution file and the project file with the same results.

Any thoughts on what I might be doing wrong?

[build_log.txt](https://github.com/seerge/g-helper/files/12450740/build_log.txt)

Regards
Russell

**Comment by seerge** [comment]
Warnings are absolutely normal :) Try to run it from the studio itself, instead of building first. Then you will see why it doesn't run.

**Reply by russellmm** [reply]
I installed VS2022 on the actual laptop and built it there. It does build and work there but the ghelper.exe is still only 250KB and it also creates a ghelper.dll.  I am selecting to build as release but the debug build does the same.

It's been a while since I have coded so I am sure there is a simple solution to having it package the .dll and other files into the ghelper.exe but I can't figure it out :)

**Reply by seerge** [reply, marked as answer]
It's in publish settings to build all dlls inside same exe

![Screenshot 2023-08-29 000209](https://github.com/seerge/g-helper/assets/5920850/60e097de-a6de-46a1-95ad-522969dde1f5)

**Reply by russellmm** [reply]
worked! Thanks.
